python/pyspark/sql/tests/connect/client/test_reattach.py

```diff
@@ -84,10 +84,10 @@ def wait_for_responses():
         # query1 and query2 could get either an:
         # OPERATION_CANCELED if it happens fast - when closing the session interrupted the queries,
         # and that error got pushed to the client buffers before the client got disconnected.
-        # OPERATION_ABANDONED if it happens slow - when closing the session interrupted the client
-        # RPCs before it pushed out the error above. The client would then get an
+        # INVALID_HANDLE.SESSION_CLOSED if it happens slow - when closing the session interrupted
+        # the client RPCs before it pushed out the error above. The client would then get an
         # INVALID_CURSOR.DISCONNECTED, which it will retry with a ReattachExecute, and then get an
-        # INVALID_HANDLE.OPERATION_ABANDONED.
+        # INVALID_HANDLE.SESSION_CLOSED.
```

Comment:
There is the same test case in https://github.com/apache/spark/blob/master/connector/connect/server/src/test/scala/org/apache/spark/sql/connect/service/SparkConnectServiceE2ESuite.scala#L66-L72
if it got flaky here, it probably can also be flaky there now.